### DISCUSSION TITLE
json5

### DISCUSSION THREAD

**Discussion by SoftSquirrel** [nlohmann/json#3206] [Q&A] [answered]
hello, when will json5 be supported

**Comment by nlohmann** [comment, marked as answer]
This is currently not planned.

**Reply by yzazik** [reply]
Why not? I'd guess, this was prompted by at least the need for comments and hexadecimals, comments are already [optionally] there and thanks to The Martian movie, it turned out that hexadecimals are human readable too. Regards.
